VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
does this intentionally have no newline at end of file? many editors automatically add this when editing